packages/route/src/Infrastructure/Doctrine/Repository/RouteRepository.php

```diff
@@ -70,7 +70,7 @@ protected function createQueryBuilder(array $filters): QueryBuilder
         if (\array_key_exists('site', $filters)) {
             $site = $filters['site'] ?? null;
             $queryBuilder->andWhere(
-                null === $site ? 'route.site IS NULL' : 'route.site = :site'
+                null === $site ? 'route.site IS NULL' : '(route.site = :site OR route.site IS NULL)'
```

Comment:
What is the use case for `site = null OR site == :site` ?

Reply:
article bundle. The request analyzer will tell us we are on site `sulu` when opening the page `https://sulu.io/blog/some-blog` but we should still get the article which is not site specific back.